.vscode/tasks.json

```diff
@@ -0,0 +1,40 @@
+{
+    "version": "2.0.0",
+    "tasks": [
+      {
+        "label": "Build mkdocs container",
+        "type": "shell",
+        "command": "docker build -t mkdocs .",
+        "options": {
+          "cwd": "${workspaceFolder}"
+        },
+        "presentation": {
+          "reveal": "always",
+          "panel": "new"
+        },
+        "problemMatcher": [],
+        "group": {
+          "kind": "build",
+          "isDefault": true
+        }
+      },
+      {
+        "label": "Run mkdocs container",
```

Comment:
ez most csak a magyart futtatja, esetleg lehetne csinálni egy másik taskot is az angolnak